src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -8716,7 +8716,6 @@ private static bool IsInvalidSubExpression(SyntaxKind kind)
                 case SyntaxKind.IfKeyword:
                 case SyntaxKind.LockKeyword:
                 case SyntaxKind.ReturnKeyword:
-                case SyntaxKind.SwitchKeyword:
```

Comment:
I didn't understand this change. How is `switch` a sub-expression?

Reply:
Didn't mean to change that. Will revert. I don't think it would change behavior, though.